imports/plugins/core/checkout/client/templates/checkout/checkout.js

```diff
@@ -15,6 +15,13 @@ Template.cartCheckout.helpers({
       return Cart.findOne();
     }
     return {};
+  },
+  cartCount() {
+    const cart = Cart.findOne();
+    if (cart.items && cart.items.length > 0) {
```

Comment:
There is a brief time when cart doesn't exist (this happens when a cart is transitioned to an order and another cart is being created for the user). In that brief time, an error is thrown in the browser console by this line.

This line will be better as:
`if (cart && cart.items && cart.items.length > 0) {`